Packs/Cybereason/Integrations/Cybereason/Cybereason.py

```diff
@@ -1910,6 +1912,66 @@ def get_sensor_id_command(client: Client, args: dict):
         return CommandResults(readable_output=f"Available Sensor IDs are {output}")
 
 
+def fetch_machine_details_command(client: Client, args: dict):
+    machine_name = str(args.get('machineName'))
+    json_body = fetch_machine_details_command_pagination_params(args)
+    response = client.cybereason_api_call('POST', '/rest/sensors/query', json_body=json_body)
+    if dict_safe_get(response, ['sensors']) == []:
+        return CommandResults(readable_output=f"Could not find any Sensor ID for the machine: {machine_name}")
+    else:
+        outputs = []
+        for single_sensor in response.get('sensors'):
+            if single_sensor.get('machineName') == machine_name:
+                outputs.append({
+                    "MachineID": single_sensor.get("sensorId"),
+                    "MachineName": single_sensor.get("machineName"),
+                    "MachineFQDN": single_sensor.get("fqdn"),
+                    "GroupID": single_sensor.get("groupId"),
+                    "GroupName": single_sensor.get("groupName")
+                })
+        empty_output_message = 'No Machine Details found for the given Machine Name'
+        return CommandResults(
+            readable_output=tableToMarkdown(
+                'Machine Details', outputs, headers=SENSOR_HEADERS) if outputs else empty_output_message,
+            outputs_prefix='Cybereason.Sensor',
+            outputs_key_field='MachineID',
+            outputs=outputs)
+
+
+def fetch_machine_details_command_pagination_params(args: dict) -> dict:
```

Comment:
Please change the function name to align with the command name.

Reply:
Done